docs/tutorials/create-waggle.md

```diff
@@ -0,0 +1,37 @@
+---
+sidebar_position: 4
+---
+
+# Building your own Waggle Device
+
+:::
+Note: This tutorial is work in progress and is made available here for early preview.
+:::
+
+Do you ever look into the sky and wonder how will the cloud move over time? Do you ever want to know if a wildfire is possibly to start in a forest? If you would like to build, design and deploy a software that could answer your questions above, then Waggle is the right choice for you. Waggle is an edge computing wireless sensor device, say, you can define what the device will be detecting by building a software on it.
+
+If you would like to gain more understandings and share your own observation, this tutorial will give you a guide of how to register your Waggle device to the Sage network. After registering, you can publish your observation and see others data on [Sage Portal](https://portal.sagecontinuum.org/apps/explore).
+
+
+## Potential Use Cases
+
+You might be ...
+- a professor that wants to introduce Waggle device to your students and plans to instruct students to make 20+ waggle devices
+- someone interested in Waggle and making an app but doesnt have access to production nodes
```

Comment:
We can probably get rid of "but ..."
How about
```
someone interested in Waggle and making apps and use the device to test the apps
```

Reply:
replace with [this comment](https://github.com/waggle-sensor/waggle-docs/pull/12#discussion_r979064070)